charts/cortx/values.yaml

```diff
@@ -84,6 +84,29 @@ externalKafka:
   adminUser: admin
   adminSecretName: kafka_admin_secret
 
+# CORTX Hare component configuration
+hare:
+  hax:
+    ports:
+      http:
+        # -- The HTTP scheme used to configure the Hax HTTP endpoint. Valid values are `http` or `https`.
+        scheme: https
+        # -- The port number of the Hax HTTP endpoint.
+        port: 22003
```

Comment:
Not really sure why this is configurable?

Reply:
I'm leaning towards removing it, since we don't allow configuration of the other internal ports (IOS, confd, etc.). If so, I'll do that in a follow up PR.